validation-test/stdlib/StringBreadcrumbs.swift

```diff
@@ -1,5 +1,3 @@
-// rdar://84233775
-// REQUIRES: rdar84233775
```

Comment:
This is passing for me now, but I would be fine with separating out reenabling it